test/functional/rpc_packages.py

```diff
@@ -377,6 +377,9 @@ def test_submitpackage(self):
         self.log.info("Submitpackage only allows packages of 1 child with its parents")
         # Chain of 3 transactions has too many generations
         legacy_pool = node.getrawmempool()
+
+        assert_raises_rpc_error(-22, "TX decode failed:", node.submitpackage, ["", ""])
+
```

Comment:
While I was going through this test after I checked out the branch in local, I realised that this assertion seems a bit out of context because of the comment on line 378. How about adding this assertion before line 378?

Reply:
Thanks! I moved it more towards the start of the function in [34c6643](https://github.com/bitcoin/bitcoin/pull/31139/commits/34c6643c2fe0e73d73edb2ddddf0900e4256de57)